index.js

```diff
@@ -137,7 +137,7 @@ module.exports.bootstrap = async ({ getPluginContext, options, refresh, setPlugi
 };
 
 function createSanityListener(client, options, refresh, getPluginContext, setPluginContext) {
-    const debouncedRefresh = debounce(refresh, 500);
+    const debouncedRefresh = debounce(refresh, 500, true);
```

Comment:
does this mean no debounce?

Reply:
I've seen a case where sanity listener is bombarded with ~20 updates at once (may be a bulk operation or something).
How about reduce the debounce to something like 10ms?

Reply:
The `debounce` implementation is here: https://github.com/stackbit/sourcebit-source-sanity/blob/master/lib/misc-util.js
unless I'm missing something it will wait for previous timeout's to finish even if `immediate=true`.

Reply:
Rodik is right and this `debounce` will throw away calls in `immediate` mode. We either need a better implementation or reduce the `debounce` time.

Reply:
in our case calling debounce <500ms is totally possible because when multiple editors edit the site, the events might come very quick one after another.

Reply:
IOW, this just shows us how being anti-lodash, and try to invent a wheel instead, eventually causes more harm than good.